lighthouse-core/lib/lh-env.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
```

Comment:
@brendankenny you have the great honor of naming this file anything you'd like :)

Reply:
@brendankenny does this work for you?

Reply:
> naming this file anything you'd like

`lighthouse-core/utilities/utils/utils.util.js` pls

Reply:
(LGTM)